docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -117,40 +120,86 @@ The Endpoint Detection Agent is easy to install on Windows or Linux systems once
     sudo systemctl status SEKOIAEndpointAgent.service
     ```
 
-Once installed, the agent collects, normalizes, and sends event logs to Sekoia.io. The protocol used to send events is HTTPS (443).
-
 #### journald configuration
 
 To get events, the agent pushes rules to the audit framework. By default, **journald**  might listen to the audit socket for events.
 
 To disable audit logging, do the following as root:
-	
-       
+
+
 	# Stop listening to audit events
 	systemctl stop systemd-journald-audit.socket 
-	
+
 	# Disable it to avoid future start
 	systemctl disable systemd-journald-audit.socket
-	
+
 	# Masking will prevent starting by other services
 	systemctl mask systemd-journald-audit.socket
-	
+
 	# Restart journald
 	systemctl restart systemd-journald
-       
-       
+
+
 A reboot may be necessary if the audit events are still appearing in the logs.
 
+=== "MacOs"
```

Comment:
Mac section appears in its own tab alone, it should be next to linux

![image](https://github.com/SEKOIA-IO/documentation/assets/679995/59db05ad-6d1d-458b-a0d2-ecdb31baceec)